jupyterhub/event-schemas/server-actions/v1.yaml

```diff
@@ -0,0 +1,59 @@
+"$id": hub.jupyter.org/server-action
+version: 1
+title: JupyterHub server events
+description: |
+  Record actions on user servers made via JupyterHub.
+
+  JupyterHub can perform various actions on user servers via 
+  direct interaction from users, or via the API. This event is 
+  recorded whenever either of those happen.
+
+  Limitations:
+
+  1. This does not record all server starts / stops, only those
+     explicitly performed by JupyterHub. For example, a user's server
+     can go down because the node it was running on dies. That will
```

Comment:
```suggestion
     can stop when its host node dies. In this situation, a stop event will not be recorded since the stop was caused by the node's failure and not an action initiated by JupyterHub. Since nodes do die in practice, a complete record of all starts and stops is not captured.
```